mmcls/models/backbones/convnext.py

```diff
@@ -220,13 +220,23 @@ def __init__(self,
                  norm_cfg=dict(type='LN2d', eps=1e-6),
                  act_cfg=dict(type='GELU'),
                  linear_pw_conv=True,
+                 use_grn=False,
```

Comment:
Please add docs for use_grn, and clarify the setting differences between V1 and V2 in docstring